webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js

```diff
@@ -65,6 +65,42 @@ export class SakaiGrader extends graderRenderingMixin(gradableDataMixin(SakaiEle
     if (typeof MathJax !== "undefined") {
       MathJax.Hub.Queue([ "Typeset", MathJax.Hub ]);
     }
+
+    window.addEventListener(
+      "message",
+      e => {
+        // Will this ever actually be a string?
+        const message = (typeof e.data === "string") ? JSON.parse(e.data) : e.data;
+        if ( message.subject !== "lti.gradeChangeNotify" ) return;
+        console.debug("The LTI Tool changed a grade - retrieving new grade");
+        console.debug(this._submission);
+        const formData = new FormData();
+        formData.valid = true;
+        formData.set("studentId", this._submission.firstSubmitterId);
+        formData.set("courseId", portal.siteId);
+        formData.set("gradableId", this.gradableId);
+        formData.set("submissionId", this._submission.id);
+        fetch("/direct/assignment/getGrade.json", {
+          method: "POST",
```

Comment:
This needs to be a GET. It makes no sense being a POST at all. You can do it like this:

https://stackoverflow.com/questions/35038857/setting-query-string-using-fetch-get-request

Reply:
Thanks - I switched to a GET but it seemed to need the cross-origin stuff to work.

          cache: "no-cache",
          credentials: "same-origin",

That leads me to wonder if the removeFeedbackAttachment bit on line 710 (approx) works.  I followed that pattern for getGrade - and it failed, I added back the cache and credentials bits and it started working again.  cc: @adrianfish